dwave/plugins/sklearn/__init__.py

```diff
@@ -13,3 +13,8 @@
 # limitations under the License.
 
 __version__ = '0.0.1'
+
+try:
```

Comment:
This breaks when I try to run unit tests locally, hence the try. Works when things are installed via pip it seems

Reply:
How are you running them locally?

Reply:
To be clear, this should not be in a try-catch.